### PR TITLE
Remove empty rows generated in migration

### PR DESCRIPTION
**Pull request by Neodelf**

### CODE REVIEW

**Comment by deivid-rodriguez** [comment]
Thanks! Do you think we can use `<%-` so we can keep our template readable as well?

File: lib/generators/active_admin/install/templates/migrations/create_active_admin_comments.rb.erb [hunk 1]
```diff
@@ -1,9 +1,6 @@
-<%
-parent_class = ActiveRecord::Migration
+<%parent_class = ActiveRecord::Migration
 parent_class = parent_class[5.0] if Rails::VERSION::MAJOR >= 5
-%>
-
-class CreateActiveAdminComments < <%= parent_class.to_s %>
+%>class CreateActiveAdminComments < <%= parent_class.to_s %>
```
Comment (timoschilling) [inline lib/generators/active_admin/install/templates/migrations/create_active_admin_comments.rb.erb]: use this style:
```erb
<%
parent_class = ActiveRecord::Migration
parent_class = parent_class[5.0] if Rails::VERSION::MAJOR >= 5
-%>
class CreateActiveAdminComments < <%= parent_class.to_s %>
```
This should produce the same result.

**Comment by Neodelf** [comment]
Thanks for the quick feedback

**Comment by deivid-rodriguez** [comment]
I'm not able to actually try this myself, but looks good. If either of you have confirmed that this works, I'm happy to see this merged!

**Pull request merged**